### PR TITLE
Polishes to follow up subscription process

### PR DESCRIPTION
**Pull request by ryancarlson**
Minor things, should be reviewed after #9

### CODE REVIEW

**Comment by ryancarlson** [comment]
@michaelharro this one is ready for review too, after #9

File: GTViewController/Classes/Custom Classes/GTInputFieldView.m [hunk 1]
```diff
@@ -14,13 +14,19 @@
...
+@end
+
 @implementation GTInputFieldView
 
 
 - (instancetype)createInputFieldFromElement:(TBXMLElement *)element withY:(CGFloat)yPos withStyle:(GTPageStyle*)style presentingView:(UIView *)presentingView {
     
     GTLabel *inputFieldLabel = nil;
-    UITextField *inputTextField = [[UITextField alloc]init];
+    self.inputTextField = [[UITextField alloc]init];
```
Comment (michaelharro) [inline GTViewController/Classes/Custom Classes/GTInputFieldView.m]: This is a small thing but could you add the space in front of the init method `[[UITextField alloc] init];` it improves readability.

**Comment by michaelharro** [comment]
👍

**Comment by ryancarlson** [comment]
Thanks for the review!

**Pull request merged**